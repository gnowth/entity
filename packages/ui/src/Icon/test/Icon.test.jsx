import 'jest-styled-components';
import renderer from 'react-test-renderer';
import React from 'react';

import UIIcon from '../Icon';

it('renders correctly', () => {
  const tree = renderer.create(
    <UIIcon name="apple" />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
