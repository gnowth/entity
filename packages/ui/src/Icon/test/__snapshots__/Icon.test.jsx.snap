// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<i
  className="icon icon-apple Icon__UIIcon-sc-10so2ko-0 ivuVRW"
  name="apple"
/>
`;
